plugins/GSdx/GSSetting.cpp

```diff
@@ -143,6 +143,8 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Increases the width of lines at higher than native resolutions. This ensures that the lines will keep the correct proportions and prevents aliasing. "
 				"Avoids empty lines on the screen in games such as Ridge Racer V, and clears FMVs obscured by a grid in games like the Silent Hill series and Dirge of Cerberus.";
 #ifdef _WIN32
+		case IDC_MEMORY_WRAPPING:
```

Comment:
You could add the enum in the `GSSetting.h` file and avoid the ifdef. (yes less work for me)